File: tiercache-reactor/src/main/java/io/tiercache/reactor/package-info.java
/**
 * Reactor API for the Tiercache two-level cache: the
 * {@link io.tiercache.reactor.ReactorTierCache} {@code Mono} facade over the
 * core's async view, and {@link io.tiercache.reactor.ReactorCacheFactory}
 * handing out the facades and exposing inbound invalidation events as cold
 * {@code Flux}es with bounded per-subscriber buffering.
 *
 * @since 0.4.0
 */
package io.tiercache.reactor;

File: tiercache-kotlin/src/main/kotlin/io/tiercache/kotlin/package-info.kt
/**
 * Kotlin coroutines API for Tiercache: [io.tiercache.kotlin.KTierCache] suspend facade over the
 * blocking `io.tiercache.TierCache`, invalidation events as a cold [kotlinx.coroutines.flow.Flow]
 * via [io.tiercache.kotlin.KTierCacheFactory.invalidationEvents], and the
 * [io.tiercache.kotlin.tierCache] configuration DSL.
 *
 * @since 0.1.0
 */
package io.tiercache.kotlin
